source/build/citrix.md

```diff
@@ -12,15 +12,15 @@ Below outlines how the proposed configuration will be configured for the Citrix
 | CTXADC01 / CTXNS | Citrix Netscaler ADC       | Citrix ADC Appliance                   |
 +------------------+----------------------------+----------------------------------------+
 | CTXSF01          | Citrix StoreFront          | Citrix Storefront                      |
-|                  |                            | Citrix Receiver                        |
-|                  |                            | IIS                                    |
+|                  |                            | | Citrix Receiver                      |
```

Comment:
![image](https://user-images.githubusercontent.com/3035178/99512726-b474b700-2981-11eb-8d6d-89da23e727a3.png)
 Is this how you want it to look? Are they sub-roles?

Reply:
I read that an extra line with put them under each other, as currently it just puts them all on the same line.